[docs] Document new scroll actions (closes #6072) (#6086)
DevExpress/testcafe: docs/articles/templates/actions/offset-options.md

> Offset options supply offset parameters for `t.scroll`, `t.scrollBy` and `t.scrollIntoView` actions.

Reviewer: how can these options 'supply' parameters?
```suggestion
Offset options apply to `t.scroll`, `t.scrollBy` and `t.scrollIntoView` actions.
```

```js
{
    offsetX: Number,
    offsetY: Number,
}
```

Parameter                      | Type    | Description                                                                                                                                                 | Default
------------------------------ | ------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------

> `offsetX`, `offsetY`           | Number  | Mouse pointer coordinates that define a point where the action is performed or started. If positive, TestCafe calculates coordinates relative to the top-left corner of the target element. If negative, they are calculated relative to the bottom-right corner. | The center of the target element.

Reviewer: very very convoluted imo. I'm not sure that even I understand this 100%. Why does it work like this? What decides if the offset applies to the action's starting point or the mouse pointer's target position? You can break it down into smaller chunks if necessary...

Author: ```suggestion
`offsetX`, `offsetY`           | Number  | A point inside of the target element. Before TestCafe performs a scroll, it reveals this point of the element. If positive, TestCafe calculates coordinates relative to the top-left corner of the target element. If negative, they are calculated relative to the bottom-right corner.  | Top left of the target element.
```

Reviewer: good enough but what do you think about this version?

```suggestion
`offsetX`, `offsetY`           | Number  | Coordinates that define the action's starting point. If positive, TestCafe calculates coordinates relative to the top-left corner of the target element. If negative, they are calculated relative to the bottom-right corner.  | Top left corner of the target element
```
